app/controllers/observations_controller/form_helpers.rb

```diff
@@ -86,7 +86,8 @@ def init_project_vars
     @project_checks = {}
     @projects.each do |proj|
       @project_checks[proj.id] = (proj.open_membership &&
-                                  proj.accepting_observations)
+                                  proj.accepting_observations &&
```

Comment:
@mo-nathan 
#### Question: Should we remove `accepting_observations`?
“Project is accepting new observations from members” is largely redundant with date ranges.
It’s hard for me to see a non-edge use case for having both.
I suggest ditching `accepting_observations`.

Reply:
Yes, we should get rid of accepting_observations.  It's now redundant.

Reply:
Done.